src/applications/disability-benefits/new-686/config/constants.js

```diff
@@ -0,0 +1,3229 @@
+    { label: 'Michoacán', value: 'michoacan' },
+    { label: 'Morelos', value: 'morelos' },
+    { label: 'Nayarit', value: 'nayarit' },
+    { label: 'Nuevo León', value: 'nuevo-leon' },
+    { label: 'Oaxaca', value: 'oaxaca' },
+    { label: 'Puebla', value: 'puebla' },
+    { label: 'Querétaro', value: 'queretaro' },
+    { label: 'Quintana Roo', value: 'quintana-roo' },
+    { label: 'San Luis Potosí', value: 'san-luis-potosi' },
+    { label: 'Sinaloa', value: 'sinaloa' },
+    { label: 'Sonora', value: 'sonora' },
+    { label: 'Tabasco', value: 'tabasco' },
+    { label: 'Tamaulipas', value: 'tamaulipas' },
+    { label: 'Tlaxcala', value: 'tlaxcala' },
+    { label: 'Veracruz', value: 'veracruz' },
+    { label: 'Yucatán', value: 'yucatan' },
+    { label: 'Zacatecas', value: 'zacatecas' },
+  ],
+  USA: states50AndDC
+    .concat([
+      { label: 'American Samoa', value: 'AS' },
+      { label: 'Armed Forces Americas (AA)', value: 'AA' },
+      { label: 'Armed Forces Europe (AE)', value: 'AE' },
+      { label: 'Armed Forces Pacific (AP)', value: 'AP' },
+      { label: 'Federated States Of Micronesia', value: 'FM' },
+      { label: 'Guam', value: 'GU' },
+      { label: 'Marshall Islands', value: 'MH' },
+      { label: 'Northern Mariana Islands', value: 'MP' },
+      { label: 'Palau', value: 'PW' },
+      { label: 'Puerto Rico', value: 'PR' },
+      { label: 'Virgin Islands', value: 'VI' },
+    ])
+    .sort((stateA, stateB) => stateA.label.localeCompare(stateB.label)),
+};
+
+const suffixes = ['Jr.', 'Sr.', 'II', 'III', 'IV'];
+
+const genders = [
+  { label: 'Female', value: 'F' },
+  { label: 'Male', value: 'M' },
+];
+
+const months = [
+  { label: 'Jan', value: 1 },
+  { label: 'Feb', value: 2 },
+  { label: 'Mar', value: 3 },
+  { label: 'Apr', value: 4 },
+  { label: 'May', value: 5 },
+  { label: 'Jun', value: 6 },
+  { label: 'Jul', value: 7 },
+  { label: 'Aug', value: 8 },
+  { label: 'Sep', value: 9 },
+  { label: 'Oct', value: 10 },
+  { label: 'Nov', value: 11 },
+  { label: 'Dec', value: 12 },
+];
+
+const twentyNineDays = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+];
+const thirtyDays = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+  '30',
+];
+const thirtyOneDays = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+  '30',
+  '31',
+];
+
+const days = {
+  1: thirtyOneDays,
+  2: twentyNineDays,
```

Comment:
Will this be a bug in non-leap-year years?

Reply:
Nope, it is set up to allow for both leap and non-leap years

Reply:
We actually don't need this at this point so I will remove it and commit